### PR TITLE
feat(showcase): add i3s explorer showcase

### PR DESCRIPTION
**Pull request by dryabinin94**
Added link to the I3S Explorer example - https://production.d3nof3l3x2sso4.amplifyapp.com/dashboard
image

### CODE REVIEW

File: website/showcase.json [hunk 1]
```diff
@@ -165,5 +165,12 @@
       "Blog post": "https://medium.com/vis-gl/rendering-a-minecraft-world-with-deck-gl-927b52c5c6db"
     },
     "description": "Rendering A Minecraft World With deck.gl."
+  },
+  {
```
Comment (ibgreen) [inline website/showcase.json:169]: Don't you want it at the top, as it is the latest / newest show case? Anyone coming back to this page probably won't look into the bottom of the list.

**Review by ibgreen — APPROVED** [review]
Change title of PR to "add i3s explorer showcase"?

@Pessimistress Worth mentioning new showcases in what's new, to get more eyes on them?

**Comment by ibgreen** [comment]
The comparison and about tabs don't work for me.

I think it would be good with a splash screen or page of docs explaining what the app does. Even though I know, the powerful features are not super obvious when just opening the app.

File: website/showcase.json (hunk 1 above)
Comment (dryabinin-actionengine) [reply website/showcase.json:169]: Yes I can change it to be first if no one minds.

File: website/showcase.json [hunk 2]
```diff
@@ -165,5 +165,12 @@
...
+    "name": "I3S Explorer",
+    "url": "https://production.d3nof3l3x2sso4.amplifyapp.com/",
```
Comment (felixpalmer) [inline website/showcase.json:171]: As the website isn't updated that often it would be better to create a better URL, this looks like a development URL.

**Comment by felixpalmer** [comment]
> The comparison and about tabs don't work for me.
> 
> I think it would be good with a splash screen or page of docs explaining what the app does. Even though I know, the powerful features are not super obvious when just opening the app.

I would echo this. It looks great, but I don't know what powerful features this has and my first impression was that it was broken as it took some time to load the buildings.

**Pull request closed without merging**